easybuild/easyconfigs/p/polars/polars-1.27.1-gfbf-2024a.eb

```diff
@@ -0,0 +1,1085 @@
+    ('utf8_iter', '1.0.4'),
+    ('uuid', '1.15.1'),
+    ('value-trait', '0.10.1'),
+    ('vcpkg', '0.2.15'),
+    ('version_check', '0.9.5'),
+    ('vsimd', '0.8.0'),
+    ('walkdir', '2.5.0'),
+    ('want', '0.3.1'),
+    ('wasi', '0.11.0+wasi-snapshot-preview1'),
+    ('wasi', '0.13.3+wasi-0.2.2'),
+    ('wasm-bindgen', '0.2.100'),
+    ('wasm-bindgen-backend', '0.2.100'),
+    ('wasm-bindgen-futures', '0.4.50'),
+    ('wasm-bindgen-macro', '0.2.100'),
+    ('wasm-bindgen-macro-support', '0.2.100'),
+    ('wasm-bindgen-shared', '0.2.100'),
+    ('wasm-streams', '0.4.2'),
+    ('web-sys', '0.3.77'),
+    ('web-time', '1.1.0'),
+    ('winapi', '0.3.9'),
+    ('winapi-i686-pc-windows-gnu', '0.4.0'),
+    ('winapi-util', '0.1.9'),
+    ('winapi-x86_64-pc-windows-gnu', '0.4.0'),
+    ('windows', '0.57.0'),
+    ('windows-core', '0.52.0'),
+    ('windows-core', '0.57.0'),
+    ('windows-implement', '0.57.0'),
+    ('windows-interface', '0.57.0'),
+    ('windows-link', '0.1.0'),
+    ('windows-registry', '0.2.0'),
+    ('windows-result', '0.1.2'),
+    ('windows-result', '0.2.0'),
+    ('windows-strings', '0.1.0'),
+    ('windows-sys', '0.52.0'),
+    ('windows-sys', '0.59.0'),
+    ('windows-targets', '0.48.5'),
+    ('windows-targets', '0.52.6'),
+    ('windows_aarch64_gnullvm', '0.48.5'),
+    ('windows_aarch64_gnullvm', '0.52.6'),
+    ('windows_aarch64_msvc', '0.48.5'),
+    ('windows_aarch64_msvc', '0.52.6'),
+    ('windows_i686_gnu', '0.48.5'),
+    ('windows_i686_gnu', '0.52.6'),
+    ('windows_i686_gnullvm', '0.52.6'),
+    ('windows_i686_msvc', '0.48.5'),
+    ('windows_i686_msvc', '0.52.6'),
+    ('windows_x86_64_gnu', '0.48.5'),
+    ('windows_x86_64_gnu', '0.52.6'),
+    ('windows_x86_64_gnullvm', '0.48.5'),
+    ('windows_x86_64_gnullvm', '0.52.6'),
+    ('windows_x86_64_msvc', '0.48.5'),
+    ('windows_x86_64_msvc', '0.52.6'),
+    ('wit-bindgen-rt', '0.33.0'),
+    ('write16', '1.0.0'),
+    ('writeable', '0.5.5'),
+    ('x11rb', '0.13.1'),
+    ('x11rb-protocol', '0.13.1'),
+    ('xmlparser', '0.13.6'),
+    ('xxhash-rust', '0.8.15'),
+    ('yoke', '0.7.5'),
+    ('yoke-derive', '0.7.5'),
+    ('zerocopy', '0.7.35'),
+    ('zerocopy-derive', '0.7.35'),
+    ('zerofrom', '0.1.6'),
+    ('zerofrom-derive', '0.1.6'),
+    ('zeroize', '1.8.1'),
+    ('zerovec', '0.10.4'),
+    ('zerovec-derive', '0.10.3'),
+    ('zlib-rs', '0.4.2'),
+    ('zstd', '0.13.3'),
+    ('zstd-safe', '7.2.3'),
+    ('zstd-sys', '2.0.14+zstd.1.5.7'),
+]
+
+# Feature `edition2024` is required
+preinstallopts = """
+for file in crates/polars*/Cargo.toml; do sed -i '1i cargo-features = ["edition2024"]\n' "$file"; done && """
+# Fix by_key.rs async block problem in polars-stream
+preinstallopts += "sed -i -e 's/let mut receive_and_pass = async || {/let mut receive_and_pass = async {/' "
+preinstallopts += "-e 's/receive_and_pass().await?;/receive_and_pass.await?;/' "
+preinstallopts += "crates/polars-stream/src/nodes/io_sinks/partition/by_key.rs && "
```

Comment:
Any reason this fix isn't done via a patch file?

Any relevant error messages we can mention in the comment?

Any reference we can add to a PR/issue that has more info on this?